ImageClassification/ILSVRC2012/VGG/readme.md

```diff
@@ -1,14 +1,19 @@
+#Overview
+
+This example VGG directory contains scripts to perform VGG training and inference using MKL backend and GPU backend
+
 ##Model
 
-Here we have ported the weights for the 16 and 19 layer VGG models from the Caffe model zoo (see [link](https://github.com/BVLC/caffe/wiki/Model-Zoo#models-used-by-the-vgg-team-in-ilsvrc-2014))
+Here we have ported the weights for the 16 and 19 layer VGG models from the Caffe model zoo (see [link](https://github.com/BVLC/caffe/wiki/Model-Zoo#models-used-by-the-vgg-team-in-ilsvrc-2014)): https://s3-us-west-1.amazonaws.com/nervana-modelzoo/VGG/VGG_D_fused_conv_bias.p for VGG_D and https://s3-us-west-1.amazonaws.com/nervana-modelzoo/VGG/VGG_E_fused_conv_bias.p for VGG_E
 
 ### Model script
-The model run script is included here [vgg_neon.py](./vgg_neon.py).  This script can easily be adapted for fine tuning this network but we have focused on inference here because a successful training protocol may require details beyond what is available from the Caffe model zoo.
+The model run scripts included here [vgg_neon_train.py] (./vgg_neon_train.py) and [vgg_neon_inference.py] (./vgg_neon_inference.py) perform training and inference respectively.  We are providing both the training and the inference script, they can be adapted for fine tuning this network but we have yet to test the training script because a successful training protocol may require details beyond what is available from the Caffe model zoo. The inference script will take the trained weight file as input: supply it with the VGG_D_fused_conv_bias.p or VGG_E_fused_conv_bias.p or trained models from running VGG training.
```

Comment:
change "[vgg_neon_train.py] (./vgg_neon_train.py)" to "[vgg_neon_train.py](./vgg_neon_train.py)", so the hyperlink will work

change "[vgg_neon_inference.py] (./vgg_neon_inference.py)" to "[vgg_neon_inferencepy](./vgg_neon_inference.py)", so the hyperlink will work